src/subcommand/wallet/restore.rs

```diff
@@ -4,11 +4,16 @@ use super::*;
 pub(crate) struct Restore {
   #[clap(help = "Restore wallet from <MNEMONIC>")]
   mnemonic: Mnemonic,
+  #[clap(long, short = 'p', help = "Use <PASSPHRASE> when deriving wallet")]
```

Comment:
```suggestion
  #[clap(long, help = "Use <PASSPHRASE> when deriving wallet")]
```
We only use long flags at the moment so let's stay consistent with the others.

Reply:
Will comply